dms/migrations/14.0.4.0.0/post-migration.py

```diff
@@ -5,12 +5,12 @@
 
 def convert_binary_fields_to_attachment(env):
     """Convert old user-provided thumbnails to attachments."""
-    column = openupgrade.get_legacy_name("custom_thumbnail")
```

Comment:
wouldn't be better to do `if openupgrade.column_exists(env.cr, "dms_file", column)` and not removing some migration commits?